jasao-frontend/src/components/popup/ChangePasswordPopup.tsx

```diff
@@ -0,0 +1,106 @@
+import { FC, useState } from "react";
+import styles from "../../styles/popup/Popup.module.scss";
+import classNames from "classnames/bind";
+import Popup, { buttonType } from "./Popup";
+
+const cx = classNames.bind(styles);
+
+interface SubmitEventArgs {
+  curPw: string;
+  newPw: string;
+}
+
+interface Props {
+  backgroundOpacity?: number;
+  width?: number;
+  buttonText?: {
+    cancel?: string;
+    confirm?: string;
+  };
+  title?: string;
+  message?: string;
+  onSubmit?: (e: SubmitEventArgs) => void;
+  onCancel?: () => void;
+  confirmCondition?: (e: SubmitEventArgs) => boolean;
+  visible?: boolean;
+  onVisibleChange?: (visible: boolean) => void;
+  button?: buttonType[];
+}
+
+const ChangePasswordPopup: FC<Props> = ({
+  title,
+  message,
+  onSubmit,
+  onCancel,
+  confirmCondition = () => true,
+  visible,
+  onVisibleChange,
+  backgroundOpacity,
+  buttonText,
+  width,
+  button,
+}) => {
+  const [curPw, setCurPw] = useState("");
+  const [newPw, setNewPw] = useState("");
+  const [newPwConfirm, setNewPwConfirm] = useState("");
+
+  const refresh = () => {
+    setCurPw("");
+    setNewPw("");
+    setNewPwConfirm("");
+  };
+
+  return (
+    <Popup
+      title={title}
+      message={message}
+      visible={visible}
+      onVisibleChange={onVisibleChange}
+      backgroundOpacity={backgroundOpacity}
+      button={button}
+      buttonText={buttonText}
+      width={width}
+      onButtonClick={(btn) => {
+        if (btn === "confirm" && onSubmit) {
+          onSubmit({ curPw, newPw });
+          refresh();
+        } else if (btn === "cancel" && onCancel) {
+          onCancel();
+          refresh();
+        }
+      }}
+      confirmCondition={() => {
+        return (
+          curPw !== "" &&
+          newPw !== "" &&
+          curPw !== newPw &&
+          newPw === newPwConfirm &&
+          confirmCondition({ curPw, newPw })
```

Comment:
This can be simplified
```ts
curPw && newPw && curPw != newPw && newPw != newPwConfirm && confirmCondition(...)
```

In this case, use ` newPw != newPwConfirm`. There is not substantial difference but, it's more fit to security.